src/xcompact3d.f90

```diff
@@ -275,7 +275,7 @@ subroutine init_xcompact3d()
         open(42,file='time_evol.dat',form='formatted')
      endif
   endif
-  if (itype==5) then
+  if (iforces==1) then
```

Comment:
The statement used to close the file unit 38 should also be modified. See https://github.com/xcompact3d/Incompact3d/blob/master/src/xcompact3d.f90#L312

Reply:
Hello,
Sorry for mixing two changes in one PR. I will open a dedicated PR for the sphere example. Also regarding the bug in the force.f90 file, if you refer to my original commit in the PR #269, there was minus sign infront of the `xmom` and `ymom` terms which got missing after the review commits from your side.  Can you please check once again ? I have been using the my local version instead of the latest commit all this time, so didn't realize it. But few days back, I tried it and got negative drag and lift coeffficient for the sphere problem due to the missing minus sign. 

Thanks,
Gaurav

Reply:
> Hello, Sorry for mixing two changes in one PR. I will open a dedicated PR for the sphere example. Also regarding the bug in the force.f90 file, if you refer to my original commit in the PR #269, there was minus sign infront of the `xmom` and `ymom` terms which got missing after the review commits from your side. Can you please check once again ? I have been using the my local version instead of the latest commit all this time, so didn't realize it. But few days back, I tried it and got negative drag and lift coeffficient for the sphere problem due to the missing minus sign.
> 
> Thanks, Gaurav

Hello, I am not sure to understand what you are talking about. The commit history of the code is public. Feel free to point to a specific modification in a specific commit.

Reply:
Sorry, I had mistaken about the minus sign being missing in the previous commits. I will revert all the commits and make a new commit with the bug fixed and results. Sorry for the confusion ! :)